text/0068-org-user-token-restriction.md

```diff
@@ -0,0 +1,42 @@
+- Start Date: 2023-01-18
+- RFC Type: feature
+- RFC PR: -
+- RFC Status: `draft`
+
+# Summary
+
+Provide an option to organization owners to disallow all user interaction with their organization that is using a user API token for authentication.
```

Comment:
The mention of "user API token" here feels overly specific / brittle to future changes.

I believe the key feature of these tokens that makes you single them out for rejection is not their type (user token) but rather the fact that they have no expiry. If that's the case, it would improve this feature to be focused on expiry lifetime, apart from token type.

Reply:
I'm targeting the user API tokens specifically here. This is where the concern is.

I specify "user" here because we also have "sentry app" API tokens that are similar, but different enough. 🙂